content/docs/guides/scim/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: SCIM 2.0 Integration
+meta_desc: This page provides an overview of how to configure any SAML 2.0 identity provider
+           with the Pulumi Console.
+menu:
+    userguides:
+        weight: 9
+        identifier: scim
+
+---
+
+The [Pulumi Console](https://app.pulumi.com) supports System for Cross-domain Identity Management (SCIM) 2.0 integration with different identity providers. This support requires Pulumi Enterprise Edition. To learn more about the capabilities of the Enterprise Edition, see the [pricing page]({{< relref "/pricing" >}}).
```

Comment:
```suggestion
The [Pulumi Console](https://app.pulumi.com) supports System for Cross-domain Identity Management (SCIM) 2.0 integration with different identity providers. SCIM enables you to manage your users and groups centrally in your Identity Provider (IdP) and then sync those users and groups to the Pulumi Console. This support requires Pulumi Enterprise Edition. To learn more about the capabilities of the Enterprise Edition, see the [pricing page]({{< relref "/pricing" >}}).
```

Not sure if we do this in user guides or not, but thought it might be helpful to articulate the value this feature provides.